tests/test_parser/test_openapi.py

```diff
@@ -13,50 +12,6 @@
 MODULE_NAME = "openapi_python_client.parser.openapi"
 
 
```

Comment:
The stuff I've removed from this file, and all the other files below, is obsolete because I now have more straightforward tests covering the same conditions.

Failure cases like the ones here are now in `end_to_end_tests/generator_errors_and_warnings`; success cases are in `end_to_end_tests/generated_code_live_tests`.